### PR TITLE
Add batteryCharging for locks

### PR DESCRIPTION
**Pull request by steffenrapp**
Adds batteryCharging for locks --> Flag indicating if the batteries of the Nuki device are charging at the moment

See: https://developer.nuki.io/page/nuki-bridge-http-api-1-13/4/#heading--list

### CODE REVIEW

File: poetry.lock [hunk 1]
```diff
@@ -0,0 +1,470 @@
...
+description = "The uncompromising code formatter."
+optional = false
+python-versions = ">=3.8"
+files = [
+    {file = "black-24.2.0-cp310-cp310-macosx_10_9_x86_64.whl", hash = "sha256:6981eae48b3b33399c8757036c7f5d48a535b962a7c2310d19361edeef64ce29"},
+    {file = "black-24.2.0-cp310-cp310-macosx_11_0_arm64.whl", hash = "sha256:d533d5e3259720fdbc1b37444491b024003e012c5173f7d06825a77508085430"},
+    {file = "black-24.2.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:61a0391772490ddfb8a693c067df1ef5227257e72b0e4108482b8d41b5aee13f"},
+    {file = "black-24.2.0-cp310-cp310-win_amd64.whl", hash = "sha256:992e451b04667116680cb88f63449267c13e1ad134f30087dec8527242e9862a"},
+    {file = "black-24.2.0-cp311-cp311-macosx_10_9_x86_64.whl", hash = "sha256:163baf4ef40e6897a2a9b83890e59141cc8c2a98f2dda5080dc15c00ee1e62cd"},
+    {file = "black-24.2.0-cp311-cp311-macosx_11_0_arm64.whl", hash = "sha256:e37c99f89929af50ffaf912454b3e3b47fd64109659026b678c091a4cd450fb2"},
```
Comment (pschmitt) [inline poetry.lock:15]: image

Comment (steffenrapp) [reply poetry.lock:15]: Sorry I don't get it :) those lines were generated automatically or is there a problem with the feature?

Comment (pschmitt) [reply poetry.lock:15]: No, I just don't want to commit the lockfile.

Comment (steffenrapp) [reply poetry.lock:15]: Not sure how it happened, I didn't do it intentionally... Gonna check how I can fix it.

Comment (pschmitt) [reply poetry.lock:15]: Don't worry about it, I got you!

#89

**Comment by pschmitt** [comment]
Superseded by #89

**Pull request closed without merging**